clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/DisableServiceAtomicOperationSpec.groovy

```diff
@@ -40,7 +43,42 @@ class DisableServiceAtomicOperationSpec extends CommonAtomicOperation {
     operation.operate([])
 
     then:
+    1 * autoscaling.describeScalableTargets(_) >> new DescribeScalableTargetsResult()
+      .withScalableTargets(new ScalableTarget()
+        .withSuspendedState(new SuspendedState()
+          .withScheduledScalingSuspended(false)
+          .withDynamicScalingInSuspended(false)
+          .withDynamicScalingOutSuspended(false)))
     1 * autoscaling.registerScalableTarget(_)
     1 * ecs.updateService(_)
   }
+
+  void 'should not suspend autoscaling if it is already suspended'() {
```

Comment:
Can you also add a check in the `then` clause to verify that registerScalableTarget api call is not invoked?